src/Integration.UnitTests/SonarAnalyzer/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -12,6 +12,7 @@
 using SonarLint.VisualStudio.Integration.Service;
 using SonarLint.VisualStudio.Integration.SonarAnalyzer;
 using System;
+using System.Linq;
 using System.Windows.Threading;
 
 namespace SonarLint.VisualStudio.Integration.UnitTests.SonarAnalyzer
```

Comment:
Can you remove the SonarAnalzer suffix? ActiveSolutionBoundTracker service is general and not just for the analyzer